src/coprocessor/metrics.rs

```diff
@@ -98,3 +98,35 @@ lazy_static! {
             20.0, 24.0, 28.0, 32.0, 48.0, 64.0, 96.0, 128.0, 192.0, 256.0]
         ).unwrap();
 }
+
+/// `ScanCounterMetrics` is for recording range query and point query.
+pub struct ScanCounterMetrics {
+    range_query: usize,
+    point_query: usize,
```

Comment:
How about exporting `range_query` as `range`, and `point_query` as `point`, I think it is straightforward and convenient.

Reply:
+1

Reply:
I prefer keeping them private because we only need to export 3 interfaces: `add_range`, `add_point` and `merge`. We can handle it more easily.